core/include/traccc/fitting/status_codes.hpp

```diff
@@ -0,0 +1,20 @@
+/** TRACCC library, part of the ACTS project (R&D line)
+ *
+ * (c) 2022-2024 CERN for the benefit of the ACTS project
```

Comment:
This could be:

```suggestion
 * (c) 2025 CERN for the benefit of the ACTS project
```

😉